cvrf2csaf/section_handlers/vulnerability.py

```diff
@@ -42,12 +113,18 @@ def _process_optional_elements(self, root_element):
             vulnerability['id'] = {'system_name': root_element.ID.attrib['SystemName'],
                                    'text': root_element.ID.text}
 
+        if hasattr(root_element, 'Involvements'):
+            vulnerability['involvements'] = self._handle_involvements(root_element.Involvements)
+
         if hasattr(root_element, 'Notes'):
             # reuse DocumentNotes handler
             notes_handler = DocumentNotes()
```

Comment:
(Was not introduced in thi change set) Or, adapt the architecture to provide a `Notes()` (and as seen further below) a `References()` handler to then mix these in where needed?

Reply:
Agree. That should be done in refactoring...

Reply:
Will do in another PR.